subprojects/enterprise/src/main/java/org/gradle/internal/enterprise/impl/DefaultGradleEnterprisePluginCheckInService.java

```diff
@@ -55,20 +57,29 @@ public DefaultGradleEnterprisePluginCheckInService(
         MINIMUM_SUPPORTED_PLUGIN_VERSION_FOR_CONFIGURATION_CACHING.getMajor(),
         MINIMUM_SUPPORTED_PLUGIN_VERSION_FOR_CONFIGURATION_CACHING.getMinor());
 
+    public static final VersionNumber MINIMUM_SUPPORTED_PLUGIN_VERSION_FOR_ISOLATED_PROJECTS = VersionNumber.version(3, 15);
```

Comment:
What about TD, PTS, and test retries? AFAIK they won't be compatible in that version.

Reply:
The plan is to do something on the GE plugin side for 3.15 to have them not fail the build with project isolation. That may work by either not applying them at all or fixing them.